config/index.js

```diff
@@ -519,12 +519,6 @@ var conf = convict({
     env: 'LOCKOUT_ENABLED',
     default: false
   },
-  newLoginNotificationEnabled: {
-    doc: 'Is the new-login notification email enabled',
-    format: Boolean,
-    env: 'NEW_LOGIN_NOTIFICATION_ENABLED',
-    default: true
-  },
```

Comment:
I opted to remove this ancient feature-flag rather than copy it into a new location.